packages/features/bookings/lib/service/RegularBookingService.ts

```diff
@@ -412,6 +414,49 @@ function formatAvailabilitySnapshot(data: {
   };
 }
 
+function buildBookingCreatedPayload({
```

Comment:
I'm hesitant about having this here in `RegularBookingService` it feels like it should live as a method in another service.

Reply:
So, this is building all the data that is needed to fire an event(bookingCreated). This is usually the responsibility of the calling site itself because it knows where to get the data from and then only thing left is to properly place it as required by event handler signature. Type safety ensures they are passed correctly.